metrics-config.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version='1.1' encoding='UTF-8'?>
```

Comment:
Perhaps it would be a good time to add a new directory for Jenkins configuration files such as this, the log properties, yaml files, etc

Reply:
Yes. Will give it some thought.